.github/workflows/test-backends.yml

```diff
@@ -22,7 +22,7 @@ jobs:
 
 
   Integration-Tests-Shared-Middle-Layer:
-    if: (github.event_name == 'workflow_dispatch') || (github.event_name == 'push' && github.ref == 'refs/heads/main')
+    if: false && (github.event_name == 'workflow_dispatch') || (github.event_name == 'push' && github.ref == 'refs/heads/main')
```

Comment:
can we remove it altogether, we can still fetch it from history if we want to re-enable it

Reply:
Do we also want to remove the "Integration-Tests-Intel", which is also disabled by the same way

Reply:
let's stick to the shared middle layer for now.